frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -1,22 +1,64 @@
 <template lang='pug'>
-.c-send
+.c-send.inputgroup(:class='{"is-editing": isEditing}')
+  .c-replying(v-if='replyingMessage')
+    i18n Replying to {{replyingTo}}:&nbsp
```

Comment:
Using Vue variables (`{{` ... `}}`) is not allowed inside of `i18n`. This rule should be added to our i18n linter... /cc @snowteamer

Use single braces + `:args` instead. Also, `&nbsp` should probably not be there either. Try this maybe?

```pug
i18n(:args='{ replyingTo, replyingMessage }') Replying to {replyingTo}: "{replyingMessage}"
```

Just double-check it does what you expect though.